docs/ref/csrf.txt

```diff
@@ -284,6 +289,10 @@ The CSRF protection is based on the following things:
    Expanding the accepted referers beyond the current host or cookie domain can
    be done with the :setting:`CSRF_TRUSTED_ORIGINS` setting.
 
+.. versionadded:: 4.0
+
+    The ``Origin`` checking described in point #4 was added.
```

Comment:
Just in case future edits move points around, it might be hard to spot this reference..

```suggestion
    ``Origin`` checking was added, as described above.
```